.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
Why is this needed? I do see the advantage that this means the spec_helper is automatically included. However, this is already done in the rails_helper by requiring it explicitly. All you have to do is therefore include the rails helper which is done in every of our test files.